restservice/src/main/java/org/itevents/controller/ServiceExceptionHandler.java

```diff
@@ -51,4 +56,18 @@
         LOGGER.error(ex.getMessage());
         return new ResponseEntity<>(ex.getMessage(), HttpStatus.UNAUTHORIZED);
     }
+
+    @ExceptionHandler(NotificationServiceException.class)
```

Comment:
@IgorMaksymov this `@ExceptionHandler` is redundant because you cannot handle it here. `MailNotificationService` is invoked by `MailSenderScheduler`